### PR TITLE
Switch to GitHub Actions

### LINKED ISSUE #56: Add GitHub Actions Workflow
And remove the `.travis.yml` and Travis related stuff.

### PR DESCRIPTION
**Pull request by tannewt**
Fixes #56

### CODE REVIEW

**Comment by tannewt** [comment]
GITHUB_TOKEN is provided by actions by default. I think we just need to add pypi_username and pypi_password as secrets.

**Comment by sommersoft** [comment]
>GITHUB_TOKEN is provided by actions by default.

Oh yeah. My brain was dragging in API token...which obviously doesn't apply here.

**Comment by siddacious** [comment]
@tannewt please update to use the new workflow files in TestRepo:
https://github.com/adafruit/Adafruit_CircuitPython_TestRepo/tree/master/.github/workflows

Please also added the updates to the build badge as shown in TestRepo:
https://github.com/adafruit/Adafruit_CircuitPython_TestRepo/blame/master/README.rst#L12

Thanks!

**Comment by tannewt** [comment]
@siddacious Done.

File: {{ cookiecutter.library_name }}/.gitignore [hunk 1]
```diff
@@ -4,7 +4,7 @@ __pycache__
 _build
 *.pyc
 .env
-build*
+build*/
```
Comment (siddacious) [inline {{ cookiecutter.library_name }}/.gitignore]: better yes, thanks

**Pull request merged**